google-cloud-firestore/lib/google/cloud/firestore/client.rb

```diff
@@ -173,6 +173,17 @@ def doc document_path
         # @param [String, DocumentReference, Array<String|DocumentReference>]
         #   docs One or more strings representing the path of the document, or
         #   document reference objects.
+        # @param [Array<String|FieldPath>] field_mask One or more field path
```

Comment:
Should `FieldPath` be fully-qualified here?

Reply:
It does not need to be. YARD finds the constant in the current namespace. This is consistent with how existing YARD docs reference `FieldPath`.